libr/core/cconfig.c

```diff
@@ -1832,12 +1880,19 @@ R_API int r_core_config_init(RCore *core) {
 
 	/* asm */
 	//asm.os needs to be first, since other asm.* depend on it
-	SETCB("asm.os", R_SYS_OS, &cb_asmos, "Select operating system (kernel) (linux, darwin, w32,..)");
+	n = NODECB("asm.os", R_SYS_OS, &cb_asmos);
+	SETDESC(n, "Select operating system (kernel)");
```

Comment:
idk, I can do, but its the every pref missing those spaces here. Should I change them all?